### ISSUE TITLE
NSBitmapImageRep tiffData is now gone in macOS 10.15

### ISSUE THREAD

**Issue by cherpake** [SDWebImage/SDWebImage#2767]
I suggest wrapping it with MAC_OS_X_VERSION_MAX_ALLOWED macro or remove it completely.

```
- (CGImageSourceRef)imageSource {
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
#if MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_14
    if (_tiffData) {
        return (__bridge CGImageSourceRef)(_tiffData);
    }
#endif
#pragma GCC diagnostic pop
    return NULL;
}

```

### New Issue Checklist

* [X] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [X] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [X] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g.  macos
 Platform Version        | e.g. 10.15.0 
 SDWebImage Version      | e.g. 5.0.0
 Integration Method      | e.g. *
 Xcode Version           | e.g. Xcode 10 / Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

**Comment by kinarob** [comment]
@cherpake We remove it completely in v5.0.6.

See: [5.0.6 - Xcode 11 Compatible](https://github.com/SDWebImage/SDWebImage/releases/tag/5.0.6)  
#2745 

Please update v5.0.6.

**Comment by cherpake** [comment]
I need to improve my searching skills :) thanks.
